### ISSUE TITLE
Add analytics/metrics logging to GraphQL API

### ISSUE THREAD

**Issue by olemp** [Puzzlepart/did#536]
- [x] I have searched for existing issues to avoid creating duplicates

### TLDR
Add analytics (Azure AI or similiar) to GraphQL API.

### Description
Add analytics (Azure AI or similiar) to GraphQL API.

Seems like [Apollo GraphQL Studio](https://studio.apollographql.com/) is the way to go. Very easy setup and automatic metrics just by adding some config:

![image](https://user-images.githubusercontent.com/7606007/93208287-a70e3700-f75c-11ea-8d0d-35eb28813525.png)

![image](https://user-images.githubusercontent.com/7606007/93208210-86de7800-f75c-11ea-9a92-e95ac74b9357.png)

We can also get automatic reports in Slack:

![image](https://user-images.githubusercontent.com/7606007/93208226-8e9e1c80-f75c-11ea-84df-090ba18a6484.png)

### Resources
- https://www.apollographql.com/docs/studio/setup-analytics/
